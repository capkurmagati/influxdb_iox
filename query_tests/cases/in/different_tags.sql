-- Tests for the correctness when chunks have different tag sets
-- IOX_SETUP: OneMeasurementTwoChunksDifferentTagSet

-- select a column that is not a key
SELECT temp from h2o;

-- select all the different tags and should be filled in a null in different sets
SELECT * from h2o;
