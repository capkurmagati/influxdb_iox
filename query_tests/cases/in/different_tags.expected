-- Test Setup: OneMeasurementTwoChunksDifferentTagSet
-- SQL: SELECT temp from h2o;
+------+
| temp |
+------+
| 70.4 |
|      |
| 53.4 |
|      |
+------+
-- SQL: SELECT * from h2o;
+--------+------------+---------+-------+------+-------------------------------+
| city   | other_temp | reading | state | temp | time                          |
+--------+------------+---------+-------+------+-------------------------------+
|        |            |         | MA    | 70.4 | 1970-01-01 00:00:00.000000050 |
|        | 70.4       |         | MA    |      | 1970-01-01 00:00:00.000000250 |
| Boston |            | 51      |       | 53.4 | 1970-01-01 00:00:00.000000050 |
| Boston | 72.4       |         |       |      | 1970-01-01 00:00:00.000000350 |
+--------+------------+---------+-------+------+-------------------------------+
